UserService/UserService.Repo/Repository.cs

```diff
@@ -671,5 +672,31 @@ public async Task<bool> DeleteUserAsync(int userId, CancellationToken cancellati
             }
             return success;
         }
+
+        public async Task<List<model.User>> GetAllUsers()
+        {
+            return _context.User.Select(x => new model.User()
+            {
+                ID = x.Id,
+                FirebaseUID = x.FirebaseUid
+            }).ToList();            
+        }
+
+        private void UpdateLastLogin(string firebaseUserID, DateTime? dateLastLogin)
+        {
+            var user = _context.User.FirstOrDefault(x => x.FirebaseUid == firebaseUserID);
+            DateTime dtChecked = DateTime.UtcNow;
```

Comment:
I'd prefer this to come from higher up the stack, probably from before the call to Firebase is made.  Otherwise, we could be (slightly) over-emphasising how recent the data is.  (And, it doesn't make sense to have different values stored against different users, who were checked at the same time.)

Reply:
agreed